client/components/serving-menu.jsx

```diff
@@ -0,0 +1,44 @@
+ServingMenu = React.createClass({
+
+  mixins: [ReactMeteorData, ReactRouter.Navigation],
+
+  getMeteorData() {
+
+    return {
+      game: Games.findOne(this.props.params.id)
+    };
+  },
+
+  onSelect(item) {
+    this.transitionTo('/game/' + this.props.params.id + '/play', item.query);
+  },
+
+  render() {
+
+    var menuItems = [
+      { title: this.data.game.player1, query: {server: 0} },
+      { title: this.data.game.player2, query: {server: 1} }
+    ];
+
+    return (
+      <div className="game-container">
+        <div className="player-container left">
+          <p className="player-name">{this.data.game.player1}</p>
+          <h1 className="score">0</h1>
```

Comment:
I know you are in progress on a lot of these items :)...just trying to understand the approach.

Will the scores become React components too?